internal/annotations/metrics/metrics.go

```diff
@@ -55,7 +55,7 @@ func MakeBundleMetadataLabels(cfg *config.Config) map[string]string {
 // to CustomResourceDefinitions and ClusterServiceVersions.
 func MakeBundleObjectAnnotations(cfg *config.Config) map[string]string {
 	return map[string]string{
-		BuilderObjectAnnotation: getSDKBuilder(sdkversion.Version),
+		BuilderObjectAnnotation: getSDKBuilder(strings.TrimSuffix(sdkversion.Version, "+git")),
```

Comment:
We should talk about this change. If users generate a bundle with a non-tagged binary, we'll think they used the previous tag.